### DISCUSSION TITLE
Can I customize the ColorColumn style?

### DISCUSSION THREAD

**Discussion by myregistration** [filamentphp/filament#12172] [Help]
### Package

Table builder

### Package Version

v3

### How can we help you?

The column is displaying an array of colors correctly, but I would like to modify the styling.  I tried using extraAttributes, but that doesn't appear to work.  I also tried adding !important and created a TailwindCSS class, but those didn't work either.  Thanks!

`Tables\Columns\ColorColumn::make('colors')->extraAttributes(['class' => 'w-4 h-4 rounded-none'])`

I added the following to app.css, but it didn't work either

```
    .fi-ta-color-item {
        @apply w-2 h-2 rounded-none !important;
    }
```

**Comment by leandrocfe** [comment]
You should create a custom theme
https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme
